src/config/theme/colors.ts

```diff
@@ -1,10 +1,27 @@
+// const colors = {
+//   background: '#262529',
+//   backgroundSecondary: '#212024',
+//   toolbar: '#111',
+//   text: '#fff',
+//   textSecondary: '#95a5a6',
+//   primary: '#badc58',
+//   muted: '#333',
+//   warning: '#f1c40f',
+//   error: '#e74c3c',
+//   success: '#2ecc71',
+//   female: '#fd79a8',
+//   male: '#74b9ff',
+// }
 const colors = {
-  background: '#262529',
-  backgroundSecondary: '#212024',
-  toolbar: '#111',
-  text: '#fff',
+  background: '#F3F3F3',
+  backgroundSecondary: '#fff',
+  toolbar: '#53B387',
+  toolbarPrimary: '#FFF',
+  toolbarSecondary: '#EAEAEA',
+  footer: '#1A2F41',
```

Comment:
Add this color as `toolbarSecondary`, since it is a toolbar but on secondary color 🤓